src/pods/test-verb-forms/test-verb-forms.component.tsx

```diff
@@ -46,6 +47,8 @@ export const TestVerbFormComponent: React.FC<Props> = props => {
     onNextQuestion();
   };
 
+  const { speak, voices } = useSpeechSynthesis();
```

Comment:
I think you have missing the culture (mine is speaking with  spanish pronunciation :))

en-uk - Daniel  

sounds quite good

https://mikeyparton.github.io/react-speech-kit/

Can you give a try and configure it?

Thanks